restapi/src/main/java/io/stargate/web/docsapi/service/query/DocumentSearchService.java

```diff
@@ -180,6 +181,16 @@
         .take(1);
   }
 
+  public Flowable<RawDocument> getDocumentTtlInfo(
```

Comment:
Java docs missing here.. Not sure if this was added or I missed this..

Reply:
Yea this existed in previous reviews, must have just missed javadoc. Will add